### PR TITLE
cppfront : add missing headers

### PR DESCRIPTION
**Pull request by yann64**
Headers inclusion is missing from original recipe.

### CODE REVIEW

File: dev-lang/cppfront/cppfront-0.7.4.recipe [hunk 1]
```diff
@@ -36,6 +36,8 @@ BUILD()
 
 INSTALL()
 {
-	mkdir -p $prefix/bin
-	cp cppfront $prefix/bin
+	mkdir -p $binDir
+	cp cppfront $binDir
```
Comment (jmairboeck) [inline dev-lang/cppfront/cppfront-0.7.4.recipe:40]: This was correct before. `$binDir` includes the secondaryArch subdir on secondary architectures (i.e. x86), which is not wanted here.

File: dev-lang/cppfront/cppfront-0.7.4.recipe [hunk 2]
```diff
@@ -36,6 +36,8 @@ BUILD()
...
+	mkdir -p $includeDir/cppfront
+	cp source/* $includeDir/cppfront/
```
Comment (jmairboeck) [inline dev-lang/cppfront/cppfront-0.7.4.recipe:42]: Shouldn't this just copy the headers and not everything?

**Review by jmairboeck — CHANGES_REQUESTED** [review]

**Comment by Begasus** [comment]
@jmairboeck good to go?

**Comment by jmairboeck** [comment]
I didn't test it yet, but I think so.

**Comment by Begasus** [comment]
OK, I'll leave it alone for now then, for the LGTM (untested also) :)

**Comment by yann64** [comment]
Hi sorry for the tardiness.
I won't be getting close to my Haiku machine before weekend at best so do not expect me updating the PR before. Sorry again 😐

**Comment by yann64** [comment]
I will need to check if copying only *.h into headers directory  (instead of everything) is ok to compile generated CPP code.

**Review by yann64 — COMMENTED** [review]
Corrected include path.
Tested generated package to compile sample source code. All works OK now.

**Review by jmairboeck — APPROVED** [review]

**Pull request merged**